docs/using-airbyte/schema-change-management.md

```diff
@@ -6,100 +6,120 @@ products: all
-| ------------------------ | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
-| New Column               | The new colummn will be created in the destination. Values for the column will be filled in for the updated rows. If you are missing values for rows not updated, a backfill can be done by completing a full resync or through the `Backfill new or renamed columns` option (see below)                                                                         |
-| Removal of column        | The old column will be removed from the destination.                                                                                                                                                                                                                                                                                                             |
-| New stream               | The first sync will create the new stream in the destination and fill all data in as if it is an initial sync.                                                                                                                                                                                                                                                   |
-| Removal of stream        | The stream will stop updating, and any existing data in the destination will remain.                                                                                                                                                                                                                                                                             |
-| Column data type changes | The data in the destination will remain the same. For those syncing on a Destinations V2 destination, any new or updated rows with incompatible data types will result in a row error in the destination tables and show an error in the `airbyte_meta` field. You will need to refresh the schema and do a full resync to ensure the data types are consistent. |
+## Detecting Schema Changes
 
-## Detect and Propagate Schema Changes
+For Cloud users, Airbyte automatically checks for any changes in your source schema immediately before syncing. For Self-hosted users, Airbyte checks for any changes in your source schema immediately before syncing, at most every 24 hours. This means your source schema changes may not be automatically propagated if it has been less than 24 hours since the previous schema check.
```

Comment:
Should this be `self-managed` or do we use `self-hosted` interchangeably?

Reply:
Fair point, editing this to self-managed